newswires/app/controllers/ViteController.scala

```diff
@@ -118,7 +125,7 @@ class ViteController(
 
   private def proxyAsset(
       resource: String,
-      request: Request[AnyContent]
+      request: UserRequest[AnyContent]
```

Comment:
I don't _think_ it's a problem to change the expected type here, but I'm not 100%..